src/gui/insEdit.cpp

```diff
@@ -4097,7 +4097,7 @@ void FurnaceGUI::drawInsEdit() {
             } else if (ins->type==DIV_INS_SU) {
               macroList.push_back(FurnaceGUIMacroDesc("Cutoff",&ins->std.ex1Macro,0,ex1Max,160,uiColors[GUI_COLOR_MACRO_OTHER]));
             } else if (ins->type==DIV_INS_ES5506) {
-              macroList.push_back(FurnaceGUIMacroDesc("Filter K1",&ins->std.ex1Macro,((ins->std.ex1Macro.mode!=1)?(-ex1Max):0),ex1Max,160,uiColors[GUI_COLOR_MACRO_OTHER],false,macroRelativeMode));
```

Comment:
> so, current filter macro behavior is:
> "Fixed" option enabled: override current filter value to macro value
> "Fixed" option disabled: current filter value = filter value determined from ES5506 tab or command ± filter macro value

I was thinking of having the "relative" option rather than "fixed", so we have absolute by default...

Reply:
Pushed a commit to show you what I mean.